lib/registries/addon/components/registries-navbar/template.hbs

```diff
@@ -1,243 +1,245 @@
-<Navbar
-    data-analytics-scope='Navbar'
-    @dark={{true}}
-    ...attributes
-    as |nav|
->
-    <nav.container>
-        {{! Left Side (Band, Service Dropdown) }}
-        <nav.section data-test-left>
-
-            <OsfLink
-                data-test-brand
-                data-analytics-name='Brand'
-                @route='home'
-                local-class='Logo'
-                aria-label={{t 'navbar.go_home'}}
-            />
-
-            <BsDropdown
-                data-test-service-list
-                data-analytics-scope='Services dropdown'
-                as |dropdown|
-            >
-                <dropdown.toggle
-                    data-test-service
-                    data-analytics-name='Toggle'
-                    @classNames={{local-class 'Service Dropdown'}}
+<div {{with-branding this.brand.currentBrand}}>
```

Comment:
I had to wrap the entire `Navbar` with a div that has the `with-branding` modifier since ember doesn't allow applying modifiers to components.

Reply:
hmm according to the guides we [should be able to apply modifiers to components](https://guides.emberjs.com/v3.15.0/components/template-lifecycle-dom-and-modifiers/#toc_modifiers-and-attributes), is that an ember bug?